tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/IR/init.cc

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 void mlir::mhlo::registerAllMhloDialects(mlir::DialectRegistry &registry) {
   // clang-format off
   registry.insert<mlir::chlo::HloClientDialect,
+                  mlir::disc_ral::RalDialect,
```

Comment:
I'm not convinced this should be registered here (but  I'm also not sure that lmhlo_gpu should be there as well).
Can you leave this out?

Reply:
Done!